tests/unit/models/reddit/test_removal_reasons.py

```diff
@@ -9,19 +11,19 @@
 class TestRemovalReason(UnitTest):
     def test_equality(self):
         reason1 = RemovalReason(
-            self.reddit, subreddit=Subreddit(self.reddit, "a"), reason_id="x"
+            self.reddit, subreddit=self.reddit.subreddit("a"), id="x"
         )
         reason2 = RemovalReason(
-            self.reddit, subreddit=Subreddit(self.reddit, "a"), reason_id="2"
+            self.reddit, subreddit=self.reddit.subreddit("a"), id="2"
         )
         reason3 = RemovalReason(
-            self.reddit, subreddit=Subreddit(self.reddit, "b"), reason_id="1"
+            self.reddit, subreddit=self.reddit.subreddit("b"), id="1"
         )
         reason4 = RemovalReason(
-            self.reddit, subreddit=Subreddit(self.reddit, "A"), reason_id="x"
+            self.reddit, subreddit=self.reddit.subreddit("A"), id="x"
         )
         reason5 = RemovalReason(
-            self.reddit, subreddit=Subreddit(self.reddit, "a"), reason_id="X"
+            self.reddit, subreddit=self.reddit.subreddit("A"), reason_id="X"
```

Comment:
Please don't change the test values here. The tests should show that existing behavior is maintained, but by changing the subreddit from `"a"` to `"A"` I cannot be sure.

Reply:
I did copy-paste the subreddit value from above, so I didn't notice that the value differed.
```suggestion
            self.reddit, subreddit=self.reddit.subreddit("a"), reason_id="X"
```